projProject/project.cpp

```diff
@@ -15,30 +15,180 @@ Optionaly for *nix enviroments
+	0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF,
+	0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF,
+	0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF,
+	0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF,
+	0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF,
+	0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF,
+	0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF,
+	0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF,
+	0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF,
+	0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF,
+	0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF,
+	0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF,
+	0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF,
+	0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF,
+	0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF, 0x00, 0x00, 0xFF, 0xFF };
+
+// bitmap data for bird
+GLubyte bird[] = {
+	0x80, 0x01,     // Row 1
+	0x40, 0x02,     // Row 2
+	0x23, 0xC4,     // Row 3
+	0x24, 0x24,     // Row 4
+	0x14, 0x28,     // Row 5
+	0x08, 0x10,     // Row 6
+	0x00, 0x00,     // Row 7
+};
+
+//******* Function Definitions *********************************************************
+void readFromFile(string fileInfo, GLubyte numbers[512][512][3]){
+	ifstream fin;
+
+	// open a file
+	fin.open(fileInfo, ios::binary);
+
+	if (!fin)
+	{
+		cerr << "Unable to open " << fileInfo;
+	}
+
+	for (int i = 0; i < 512; i++)
+	{
+		for (int j = 0; j < 512; j++)
+		{
+			for (int k = 0; k < 3; k++)
+			{
+				fin.read((char*)&numbers[i][j][k], sizeof(GLubyte));
+			}
+		}
+	}
+
+	// deallocates memory for fin
+	fin.close();
+}
+
+// Draws background image
+void drawBackground(){
+	string fileName = "C:\\TEMP\\savedImg.bin";
+
+	// reads the binary file
+	readFromFile(fileName, pixelMap);
+
+	//sets the raster position
+	glRasterPos2f(-250, -250);
+
+	// draws the pixels
+	glDrawPixels(width, height, GL_RGB, GL_UNSIGNED_BYTE, pixelMap);
+}
+
+// Draws a house
+void drawHouse() {
+	glColor3f(0.502, 0.000, 0.000);			//maroon
+	glPolygonMode(GL_FRONT_AND_BACK, GL_FILL);
+	drawPolygon();
+	glEnable(GL_POLYGON_STIPPLE);				//Enbles Polygon stipple
+	glColor3f(0.5, 0.35, 0.05);
+	glPolygonStipple(pattern1);
+	drawPolygon();
+	glDisable(GL_POLYGON_STIPPLE);
+}
+void drawPolygon() {
+	glBegin(GL_POLYGON);
+	glVertex2i(-200, -200);         // Bottom left
+	glVertex2i(-200, 100);           // Top left
+	glVertex2i(200, 100);            // Top right
+	glVertex2i(200, -200);          // Bottom right
+	glEnd();
+}
+
 
 // Draws a door
 void drawDoor() {
-    glLineWidth(2.5);
+	//Draws frame for the door
     glColor3f(0, 0, 0);
-    glBegin(GL_LINE_LOOP);
-    glVertex2i(-25, -300);
-    glVertex2i(-25, -225);
-    glVertex2i(25, -225);
-    glVertex2i(25, -300);
+    glBegin(GL_LINE_STRIP);
+	glVertex2i(-50, -200);
+	glVertex2i(-50, -60);
+	glVertex2i(50, -60);
+	glVertex2i(50, -200);
```

Comment:
All the way to here. These do not need the indent

Reply:
I will remove the indentation